### ISSUE TITLE
Cannot login to dashboard in context of improperly setup service

### ISSUE THREAD

**Issue by yuklia** [serverless/serverless#8006]
1 do `sls login` and get
 Serverless plugin "serverless-python-requirements" not found. Make sure it's installed and listed in the "plugins" section of your serverless config file

2 I try `serverless plugin install serverless-python-requirements`
and get Error: {"errorMessage":"You're not authorized to access this resource. - Please contact support and provide this identifier to reference this issue - 0WS7K72V17SY"}

<details>
<summary><code>serverless.yml</code></summary>

```yaml
service: REDACTED
app: REDACTED
org: REDACTED

# You can pin your service to only deploy with a specific Serverless version
# Check out our docs for more details
# frameworkVersion: "=X.X.X"

provider:
  name: aws
  runtime: python3.8
  stage: ${opt:stage,'dev'}
  # ! You can publish messages only to topics and endpoints in the same AWS Region.
  region: ${opt:region, 'eu-west-3'}

  iamRoleStatements:
    - Effect: "Allow"
      Action:
        - "sns:*"
      Resource: "*"

    - Effect: "Allow"
      Action:
        - "ec2:*"
      Resource: "*"

plugins:
  - serverless-python-requirements
custom:
  pythonRequirements:
    dockerizePip: true
  
  snsTopicCopySnap: "${self:service}-${self:provider.stage}-sns-copy-snap"
  snsTopicCopySnapArn:
    dev:
      { "Fn::Join" : ["", ["arn:aws:sns:${self:provider.region}:", { "Ref" : "AWS::AccountId" }, ":${self:custom.snsTopicCopySnap}" ] ]  }
 
  snsTopicCreateVolume: "${self:service}-${self:provider.stage}-sns-create-volume"
  snsTopicCreateVolumeArn:
    dev:
      { "Fn::Join" : ["", ["arn:aws:sns:${self:provider.region}:", { "Ref" : "AWS::AccountId" }, ":${self:custom.snsTopicCreateVolume}" ] ]  }
  
  snsTopicDetachVolume: "${self:service}-${self:provider.stage}-sns-detach-volume"
  snsTopicDetachVolumeArn:
    dev:
      { "Fn::Join" : ["", ["arn:aws:sns:${self:provider.region}:", { "Ref" : "AWS::AccountId" }, ":${self:custom.snsTopicDetachVolume}" ] ]  }

  snsTopicAttachVolume: "${self:service}-${self:provider.stage}-sns-attach-volume"
  snsTopicAttachVolumeArn:
    dev:
      { "Fn::Join" : ["", ["arn:aws:sns:${self:provider.region}:", { "Ref" : "AWS::AccountId" }, ":${self:custom.snsTopicAttachVolume}" ] ]  }
functions:
  REDACTED:
    handler: functions/REDACTED.handler
    #If you send a message to a topic, Amazon SNS delivers the message to each endpoint that is subscribed to the topic. 
    events:
      - http:
          path: /snapshot/for/{env}
          method: get
          request:
            parameters:
              paths:
                env: true
    environment:
      snsTopicCopySnapArn: ${self:custom.snsTopicCopySnapArn.${self:provider.stage}}
            
  waitUntilCompleted:
    timeout: 600 # optional, in seconds, default is 6
    handler: functions/waitUntilCompleted.handler
     #If you send a message to a topic, Amazon SNS delivers the message to each endpoint that is subscribed to the topic. 
    events:
      - sns: ${self:custom.snsTopicCopySnap}
    environment:
      snsTopicCreateVolumeArn: ${self:custom.snsTopicCreateVolumeArn.${self:provider.stage}}
```

</details>

<details>
<summary><b><code>⚠️⚠️ REPLACE WITH FULL COMMAND NAME</code> output</b></summary>

```
Error: {"errorMessage":"You're not authorized to access this resource. - Please contact support and provide this identifier to reference this issue - 0WS7K72V17SY"}
      at _callee$ (/Users/yulia/.nvm/versions/node/v12.9.0/lib/node_modules/serverless/node_modules/@serverless/platform-sdk/src/utils/checkHttpResponse.js:19:9)
      at tryCatch (/Users/yulia/.nvm/versions/node/v12.9.0/lib/node_modules/serverless/node_modules/regenerator-runtime/runtime.js:45:40)
      at Generator.invoke [as _invoke] (/Users/yulia/.nvm/versions/node/v12.9.0/lib/node_modules/serverless/node_modules/regenerator-runtime/runtime.js:274:22)
      at Generator.prototype.<computed> [as next] (/Users/yulia/.nvm/versions/node/v12.9.0/lib/node_modules/serverless/node_modules/regenerator-runtime/runtime.js:97:21)
      at step (/Users/yulia/.nvm/versions/node/v12.9.0/lib/node_modules/serverless/node_modules/@serverless/platform-sdk/dist/utils/checkHttpResponse.js:7:191)
      at /Users/yulia/.nvm/versions/node/v12.9.0/lib/node_modules/serverless/node_modules/@serverless/platform-sdk/dist/utils/checkHttpResponse.js:7:361
      at processTicksAndRejections (internal/process/task_queues.js:85:5)

```

</details>

<b>Installed version</b>

```
Your Environment Information ---------------------------
     Operating System:          darwin
     Node Version:              12.9.0
     Framework Version:         1.73.1
     Plugin Version:            3.6.13
     SDK Version:               2.3.1
     Components Version:        2.31.2
```

**Comment by medikoo** [comment]
@yuklia great thanks for reporting, we'll look into it in next days

**Comment by yuklia** [comment]
@meagonqz 
it seems an issue related to `serverless-python-requirements` plugin so I'll redirect an issue to https://github.com/UnitedIncome/serverless-python-requirements
plugins:
  - serverless-python-requirements

**Comment by medikoo** [comment]
> When I've commented these lines ⬇️ I was able to sls login

@yuklia I believe it worked, as Framework no longer was searching for that error. Note that in main description it's _plugin not found_ error which prevents you from successfully login, while `sls login` should not dependent in anyway on current state of current service

**Comment by medikoo** [comment]
Fixed with #8056 and https://github.com/serverless/enterprise-plugin/pull/465
